.travis.yml

```diff
@@ -24,6 +24,28 @@ matrix:
       env: TOXENV=pep8py2
     - python: 3.6
       env: TOXENV=pep8py3
+    - python: 3.6
+      env: TOXENV=py36
+      apt:
+        packages:
+          - git
+          - gcc
+          - libssl-dev
+          - gnutls-bin
+      before_script:
+        - wget https://dist.opendnssec.org/source/softhsm-2.5.0.tar.gz \
+          && tar -xzf softhsm-2.5.0.tar.gz \
+          && cd softhsm-2.5.0 \
+          && ./configure --prefix=/usr \
+          && make -j$(nproc) \
+          && make install \
+          && cd .. \
+          && git clone https://github.com/stefanberger/python-gnutls.git \
```

Comment:
This shouldn't be needed, it should be a test dependency pulled via pip or similar.